### PR TITLE
Use rule identifier as check name

### PR DESCRIPTION
**Pull request by pbrisbin**
Instead of "net.csslint.Disallow@import" (source, which is a munged rule.name),
just use "import" (rule.id).

/cc @codeclimate/review

### CODE REVIEW

File: lib/cc/engine/csslint.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def run
               next unless node.name == "error"
 
               lint = node.attributes
-              check_name = lint["source"].value
+              check_name = lint["identifier"].value
```
Comment (GordonDiggs) [inline lib/cc/engine/csslint.rb]: should this be prefixed in the resulting json?

Comment (pbrisbin) [inline lib/cc/engine/csslint.rb]: Nope. Any check-prefixing is meant to be done from the outside. RuboCop's current behavior, notwithstanding, that's the future we want to build for -- so says @brynary anyway.

**Comment by pbrisbin** [comment]
@GordonDiggs you good on this?

**Comment by GordonDiggs** [comment]
LGTM

**Pull request merged**